### DISCUSSION TITLE
Sticky posts

### DISCUSSION THREAD

**Discussion by LITUATUI** [hbstack/theme#410] [Q&A] [answered]
Is it possible to make sticky posts like in [HBS](https://hbs.razonyang.com/v1/en/)?

I'm trying to configure this theme to look similar to HBS, since I think it looks great for news blogs.

**Comment by razonyang** [comment]
What does the sticky posts mean?

> I'm trying to configure this theme to look similar to HBS, since I think it looks great for news blogs.

It's hard to do that without customizating the templates.

**Comment by LITUATUI** [comment]
A Sticky Post is a post placed at the top of the front page of posts. Maybe I should say pinned post instead.

Just like the two first posts on the [HBS demo](https://hbs.razonyang.com/v1/en/).

I was slowly converting my Wordpress blog to HBS, but stopped since you recommended new users to use this theme instead.

Do you think I can manage to make this new theme to look like this? Or should I go back to HBS and finalize my conversion from WP to Hugo?


![HBS](https://github.com/hbstack/theme/assets/49299328/eb3aca88-e0ea-48a6-a376-f21c0a8514e4)

**Reply by razonyang** [reply, marked as answer]
> A Sticky Post is a post placed at the top of the front page of posts. Maybe I should say pinned post instead.

Currently, there isn't an option to make posts pinned like the previous theme did.

> Do you think I can manage to make this new theme to look like this? Or should I go back to HBS and finalize my conversion from WP to Hugo?

Yes, it's possible if you're fimilar to Hugo templates and Bootstrap. If you like the previous theme design heavily and already have a lot of converted content, I won't recommend switching the theme, since they aren't compatible and with brand new design.
